test/cutting/test_cutting_evaluation.py

```diff
@@ -315,16 +316,3 @@ def test_execute_experiments(self):
                 e_info.value.args[0]
                 == "Circuits input to execute_experiments should contain no classical registers or bits."
             )
-
-    def test_workflow_with_unused_qubits(self):
```

Comment:
Moved to `test/cutting/test_cutting_workflows.py` since it no longer calls `execute_experiments`.